docs/content/policy-language.md

```diff
@@ -1303,6 +1323,61 @@ behaviour of other rules.
 For using the `some` keyword with iteration, see
 [the documentation of the `in` operator](#membership-and-iteration-in).
 
+## Every Keyword
+
+```live:eg/every0:module:merge_down
+import future.keywords.every
+
+p {
```

Comment:
Can we reframe these examples in terms of the example data used throughout this doc?